### ISSUE TITLE
Error on buy/sell

### ISSUE THREAD

**Issue by Duchess1** [stabacco/stake-python#104]
Hi team, 

First off- thank you so much for your hard work on this brilliant library!

I’ve been using it for a while now, and recently I’ve noticed I’m getting a bad request for buy and sell orders. 
It should be noted that other functionality seems to work fine- getting current orders, getting market data, etc. 

Could be a change to the endpoint? 

```
raise ClientResponseError(
aiohttp.client_exceptions.ClientResponseError: 400, message='Bad Request', url=URL('https://global-prd-api.hellostake.com/api/sellorders')
```

**Comment by stabacco** [comment]
Thanks for reporting this, I'll take a look in the next couple of days.
Cheers

**Comment by stabacco** [comment]
closed via #107
